### PR TITLE
fix(discover) Improve table resizing behavior

### PR DESCRIPTION
**Pull request by markstory**
Improve the resizing of the results table in a few ways:

* The last column is now always set to auto-fill. This prevents
  underflow situations where a query with narrow columns is loaded into
  a wider screen. It also prevents results from being resized so that
  they don't fill 100% of the available space.
* Fixed the active state of the resize bar in firefox. Preventing the
  default there stops the `:active` style from being applied.
* Made the resize bar hit area 1px wider on each side.
* Double clicking the resize handle will convert a column to auto-fill
  making it easier to get back to an auto layout table.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/gridEditable/index.tsx [hunk 1]
```diff
@@ -319,6 +339,10 @@ class GridEditable<
       return `${COL_WIDTH_MINIMUM}px`;
     });
 
+    // The last column has no resizer and should always be a flexible column
+    // to prevent underflows.
+    widths[widths.length - 1] = `minmax(${COL_WIDTH_MINIMUM}px, auto)`;
```
Comment (leedongwei) [inline src/sentry/static/sentry/app/components/gridEditable/index.tsx]: There's always a column on the table, but imo it'll be a good idea to assert that `widths.length - 1 >= 0` to stop quirky situations from breaking the world.

**Comment by wmak** [comment]
Hm not sure if I'm using it wrong but I thought double clicking would min width a column such that you can still see content.

File: src/sentry/static/sentry/app/components/gridEditable/index.tsx (hunk 1 above)
Comment (iProgramStuff) [reply src/sentry/static/sentry/app/components/gridEditable/index.tsx]: People don't realize the world saving work being done here.

**Comment by markstory** [comment]
Right now the double click uses the same minmax(50px, auto) behavior we use for our 'undefined' width columns. This value makes the column a minimum of 50px and a max of auto, which is often longer than fit-content which is what I think you're describing.

**Review by wmak — APPROVED** [review]
LGTM :+1:

**Pull request merged**